### PR TITLE
deprecated timber/loader/paths in favor of timber/locations

### PR DESCRIPTION
**Pull request by romainmenke**
fixes : https://github.com/timber/timber/issues/1837

-------

Feedback welcome!

### CODE REVIEW

File: lib/Loader.php [hunk 1]
```diff
@@ -272,8 +276,8 @@ public function get_loader() {
 				}, $path_locations );
 			} else {
 				Helper::deprecated(
-					'add_filter( \'timber/loader/paths\', [\'path/to/my/templates\'] ) in a non-associative array',
-					'add_filter( \'timber/loader/paths\', [ 0 => [ \'path/to/my/templates\' ] ] )',
+					'add_filter( \'timber/locations\', [\'path/to/my/templates\'] ) in a non-associative array',
+					'add_filter( \'timber/locations\', [ 0 => [ \'path/to/my/templates\' ] ] )',
```
Comment (jarednova) [inline lib/Loader.php]: If I understand this deprecation notice correctly — it's warning that the arguments for `timber/locations` has changed. While this is true for `timber/loader/paths`, I don't believe it's needed

Comment (romainmenke) [reply lib/Loader.php]: True, and possibly related to the tests still failing. Still catching up on how the tests check for deprecations.

**Comment by romainmenke** [comment]
Will have more time this weekend to finish this up.

**Comment by romainmenke** [comment]
@jarednova figured out `@expectedDeprecated`
This PR should be ok now.

**Comment by gchtr** [comment]
We’re waiting for @jarednova’s final approval for the changes he requested.

**Comment by jarednova** [comment]
I approve! good shit. thanks @romainmenke!

**Pull request merged**